### ISSUE TITLE
Alarm is difficult to read in dark theme

### ISSUE THREAD

**Issue by lbouriez** [dwainscheeren/dwains-lovelace-dashboard#367]
See the picture below
![image](https://user-images.githubusercontent.com/4749766/148456556-15442373-d70f-4f56-b6fa-7e564b8feeb1.png)

It is even worst on mobile.

**Comment by lbouriez** [comment]
Seems to be this color the issue:
```
.mdc-button--outlined:not(:disabled) {
    color: var(--mdc-theme-primary, #6200ee);
}
```
If it could be a bit more white... would be amazing :)

**Comment by thebatfink** [comment]
Same issue here. You would assume it wasn’t intentional as its both difficult to see and makes them look like inactive buttons.

**Comment by dwainscheeren** [comment]
Will be fixed in 3.0 (I think)
